cmd/build.go

```diff
@@ -191,91 +240,104 @@ func runBuild(cmd *cobra.Command, args []string) {
 		return
 	}
 
-	expandedTargets := map[string]struct{}{}
-	for _, target := range targets {
-		if !strings.HasSuffix(target, "...") {
-			if _, exists := genOutput.Targets[target]; !exists {
-				log.Fatal("Target '%s' does not exist.\n", target)
-			}
-			expandedTargets[target] = struct{}{}
-			continue
+	if len(targets) > 0 {
+		ninjaArgs := []string{}
+		if log.Verbose {
+			ninjaArgs = []string{"-v", "-d", "explain"}
 		}
 
-		targetPrefix := strings.TrimSuffix(target, "...")
-		found := false
-		for availableTarget := range genOutput.Targets {
-			if strings.HasPrefix(availableTarget, targetPrefix) {
-				found = true
-				expandedTargets[availableTarget] = struct{}{}
+		for _, target := range targets {
+			if runTargets {
+				ninjaArgs = append(ninjaArgs, target+"#run")
```

Comment:
I don't really like how this just happens to be the same suffix that dbt-rules uses, but I guess is similar to how the in/out jsons have to be duplicated.

Reply:
I think there in nothing we can do about that, given that dbt and dbt-rules are separate repositories